Makefile

```diff
@@ -6,13 +6,13 @@
 #    By: aulicna <aulicna@student.42.fr>            +#+  +:+       +#+         #
 #                                                 +#+#+#+#+#+   +#+            #
 #    Created: 2023/11/23 14:30:35 by aulicna           #+#    #+#              #
-#    Updated: 2024/01/12 08:53:44 by aulicna          ###   ########.fr        #
+#    Updated: 2024/01/17 19:52:49 by aulicna          ###   ########.fr        #
 #                                                                              #
 # **************************************************************************** #
 
 NAME = minishell
 
-SRC = src/debug/main_a.c\
+SRC = src/debug/main.c\
```

Comment:
Moved everything into one main, the execve main is commented out at the end of the file.